Testing/VelaptorTests/Helpers/TestHelpers.cs

```diff
@@ -189,6 +190,69 @@ public static ImageData ToImageData(this Image<Rgba32> image)
             }
         }
 
+        /// <summary>
+        /// Returns the color of all of the pixels in the given <paramref name="row"/>.
+        /// </summary>
+        /// <param name="image">The image data that contains the row.</param>
+        /// <param name="row">The row of pixels to return.</param>
+        /// <returns>The list of row pixel colors.</returns>
+        /// <remarks>The row is 0 index based.</remarks>
+        public static IEnumerable<NETColor> GetRow(ImageData image, uint row) => GetRow(image, row, 0, image.Width);
+
+        /// <summary>
+        /// Returns the color of all of the pixels in the given <paramref name="row"/>.
+        /// </summary>
+        /// <param name="image">The image data that contains the row.</param>
+        /// <param name="row">The row of pixels to return.</param>
+        /// <param name="colStart">The column in the row to start at.</param>
+        /// <param name="colStop">The column in the row to stop at.</param>
+        /// <returns>The list of row pixel colors.</returns>
+        /// <remarks>The row is 0 index based.</remarks>
```

Comment:
Change the remarks XML comment to the suggested value below.

```suggestion
        /// <remarks>The row, colStart, and colStop are 0 index based.</remarks>
```